app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,39 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+   <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name %>
+  </div>
+
+   <div class="field">
+    <%= f.label :bio %><br />
+    <%= f.text_field :bio %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password_confirmation %><br />
+    <%= f.password_field :password_confirmation, autocomplete: "new-password" %>
+  </div>
+
+  <div class="actions">
+    <%= f.submit "Sign up" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] Ensure your sign-up page matches the design. As you can see it is somewhat centered, but your own is on the left part of the page.

  ![kore](https://user-images.githubusercontent.com/81468675/196958342-c0dd924d-8a10-48ae-a1a8-926d1aa8cd49.jpg)

- [ ] Redesign your login page as well to match the design.

  ![kore](https://user-images.githubusercontent.com/81468675/196958468-7fbfbbba-fb12-4bb7-a71f-82834ea13711.jpg)

Reply:
can you please advise how I can generate only login view?